source/features/center-reactions-popup.css

```diff
@@ -53,6 +53,6 @@ td.js-line-comments:last-child .timeline-comment-actions .add-reaction-popover::
 
 /* Keep colored button when reaction popup is open */
 .dropdown-details[open] .timeline-comment-action {
-	opacity: 1;
+	opacity: 100%;
```

Comment:
I've never seen this used anywhere before. Seems unnecessar(il)y (longer)

Reply:
It's the modern syntax and more readable as it makes it clear what the range is.

https://twitter.com/mathias/status/1253242715304857601

Reply:
![hqdefault](https://user-images.githubusercontent.com/1402241/83631214-6fa8bf80-a59d-11ea-83b5-1f608a73ff4a.jpg)